web_external/js/views/widgets/WmsLayersListWidget.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
remove this use strict

This is probably very hard to see, but you don't need to include `use 'strict';`.  It will be done automatically by the uglification process when all of the JS source in Minerva is combined into a single minerva.min.js .

I had to do a bit of digging on this myself to track it down.